securitycenter/snippets_v2/snippets_bigquery_export_v2_test.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# TODO(developer): Replace these variables before running the sample.
+import os
+import re
+import uuid
+
+from _pytest.capture import CaptureFixture
+import backoff
+from google.api_core.exceptions import InternalServerError, NotFound, ServiceUnavailable
+import pytest
+
+import snippets_bigquery_export_v2
+
+PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
+GOOGLE_APPLICATION_CREDENTIALS = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+BIGQUERY_DATASET_ID = f"sampledataset{str(uuid.uuid4()).split('-')[0]}"
+LOCATION_ID = os.environ["GCLOUD_LOCATION"]
+
+
+@pytest.fixture(scope="module")
+def bigquery_export_id():
+    bigquery_export_id = f"default-{str(uuid.uuid4()).split('-')[0]}"
+
+    create_bigquery_dataset(BIGQUERY_DATASET_ID)
+    export_filter = 'severity="LOW" OR severity="MEDIUM"'
+    snippets_bigquery_export_v2.create_bigquery_export(
+        f"projects/{PROJECT_ID}/locations/{LOCATION_ID}", export_filter, f"projects/{PROJECT_ID}/datasets/{BIGQUERY_DATASET_ID}", bigquery_export_id
+    )
+
+    yield bigquery_export_id
+
+    snippets_bigquery_export_v2.delete_bigquery_export(
+        f"projects/{PROJECT_ID}/locations/{LOCATION_ID}", bigquery_export_id
+    )
+    delete_bigquery_dataset(BIGQUERY_DATASET_ID)
```

Comment:
Assert the deletion here.

Reply:
We  are not getting any response for deletion.

Reply:
That is correct. Assert failure when trying to retrieve export config.

Reply:
This is not a test case, Pre-requisite for the below test case and in the actual delete test case we are making the assertion.